media/css/firefox/private_browsing/private-browsing-conditionals.less

```diff
@@ -0,0 +1,170 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import "../../sandstone/lib.less";
+
+.download-buttons  {
+    > div {
+        display: none;
+    }
+
+}
+/*-------------------------------------------------------------------------*/
+// Android Links and Download hide/show
+
+.android {
+    .footer .android-link {
+        display: block;
+    }
+    .footer .desktop-link {
+        display: none;
+    }
+    .download-buttons {
+    }
+    &.non-firefox .download-buttons .non-firefox-cta {
+        .desktop-download-button {
+            display: none;
+        }
+        h2 {
+            width: 100%;
+        }
+        .android-download-button {
+            display: block;
+        }
+    }
+}
+
+/*-------------------------------------------------------------------------*/
+// iOS Links and Download hide/show
+
+.ios {
+    .footer .ios-link {
+        display: block;
+    }
+    .footer .desktop-link {
+        display: none;
+    }
+
+    &.non-firefox .download-buttons .non-firefox-cta {
+        & .desktop-download-button {
+            display: none;
+        }
+        .ios-download-button {
+            display: block;
+        }
+    }
+}
+
+    
+/*-------------------------------------------------------------------------*/
+// Firefox Up to date download buttons show/hide
+
+.firefox-up-to-date {
+    .download-buttons .firefox-latest-cta {
+        display: block;
+    } 
+}
+
+
+/*-------------------------------------------------------------------------*/
+// Firefox Old download buttons show/hide plus styling
+
+.firefox-old {
+    .download-buttons .firefox-old-cta {
+       display: block; 
+    }
+    .cta {
+        .container {
+            padding: 60px 20px;
+        }
+        h2 {
+            .font-size(30px);
+            line-height: 1.4;
+            width: 440px;
```

Comment:
The design had the sentence break specifically in those spots for desktop. Not sure if I should be changing that? I can push current to demo and send link to design for confirmation on this.